tests/requirements/py3.txt

```diff
@@ -4,10 +4,10 @@ argon2-cffi >= 19.2.0
 bcrypt
 black
 docutils >= 0.19
-geoip2; python_version < '3.12'
+geoip2
 jinja2 >= 2.11.0
-numpy; python_version < '3.12'
-Pillow >= 6.2.1; sys.platform != 'win32' or python_version < '3.12'
+numpy
+Pillow >= 6.2.1; sys.platform != 'win32'
```

Comment:
`sys.platform != 'win32'` should be removed as well?

Previously it will install on Windows if Python version is < 3.12 and now it won't install on Windows no matter the python version.

Reply:
True, updated. Thanks :+1: